benefits/core/templates/core/widgets/radio_select.html

```diff
@@ -0,0 +1,19 @@
+{% with id=widget.attrs.id %}
+<div{% if id %} id="{{ id }}"{% endif %} class="radio-container">
+  {% for group, options, index in widget.optgroups %}
```

Comment:
I see a comment in the `RadioSelect` widget class: 

> `# this implementation does not support groups from ChoiceWidget.optgroups`

So do we need this loop here?

Reply:
Thanks for the explanation and deep dive into Django form widgets @angela-tran! We aren't supporting groups but this is how to loop over all options for display.